config/jobs/kubernetes-csi/csi-driver-windows-poc/csi-driver-windows-poc-config.yaml

```diff
@@ -19,19 +20,15 @@ presubmits:
       - image: gcr.io/k8s-testimages/kubekins-e2e:v20200507-2ed1a50-master
         command:
         - runner.sh
+        - /workspace/scenarios/kubernetes_e2e.py
         args:
-        - bash
-        - -c
-        - >-
-          cd $(GOPATH)/src/k8s.io/kubernetes &&
-          /workspace/kubetest
-          - --test
-          - --up
-          - --down
-          - --dump=$(ARTIFACTS)
-          # gcp-specific test args
-          - --provider=gce &&
-          /go/src/github.com/kubernetes-csi/csi-driver-host-path/deploy/util/deploy.sh
+        - --check-leaked-resources
+        - --cluster=
+        - --gcp-zone=us-west1-b
+        - --extract=ci/latest
+        - --provider=gce
+        - --gcp-nodes=2
+        - --test-cmd=../deploy/util/deploy.sh
```

Comment:
I think you'll also need:
```
--test=false
--timeout=**
```

Reply:
what --test=false mean?

Reply:
use --test-cmd, instead of the default ginkgo tester I think. That's set in the windows job so I think you'll also need that.